docs/doc_v1.0/Build_documentation.md

```diff
@@ -294,9 +378,8 @@ simulation.
 
 ### Outbreak
 
-The outbreak class will be handled in much more detail later on in
-\<\<Section XYZ\>\>. This section will focus purely on the
-initialisation.
+The outbreak class will be handled in much more detail later on. 
```

Comment:
If this is true, it's not explicitly clear that it has been covered. However, I may have just missed it.